README.md

```diff
@@ -5,10 +5,57 @@
 
  A wiki that contain all osu! rulesets
 
+## Domain
+
+- Dev build (Some function in dark mode like color invert will not work.) : https://dev.rulesets.info/
```

Comment:
This should be:
Dev build (Some functions in dark mode like color invertion might not work.) : https://dev.rulesets.info/